lib/ansible/modules/cloud/azure/azure_rm_gallery.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, (@zikalino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_gallery
+version_added: '2.9'
+short_description: Manage Azure Gallery instance.
+description:
+  - 'Create, update and delete instance of Azure Gallery.'
+options:
+  resource_group:
+    description:
+      - The name of the resource group.
+    required: true
+  name:
+    description:
+      - >-
```

Comment:
?

Reply:
`>-` is a YAML folded string with block chomping enabled (see https://yaml.org/spec/1.1/#id927557)